src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -14,7 +14,7 @@ import { CommandListArgs } from '../list/list.js';
 
 const SHEET_MIN_X = MapSheet.origin.x + 4 * MapSheet.width; // The minimum x coordinate of a valid sheet / tile
 const SHEET_MAX_X = MapSheet.origin.x + 46 * MapSheet.width; // The maximum x coordinate of a valid sheet / tile
-const SHEET_MIN_Y = MapSheet.origin.y - 41 * MapSheet.height; // The minimum y coordinate of a valid sheet / tile
+const SHEET_MIN_Y = MapSheet.origin.y - 42 * MapSheet.height; // The minimum y coordinate of a valid sheet / tile
```

Comment:
Now that's a magic number if ever I saw one. Did a test failure necessitate this? This should probably be explained in more detail, or be refactored out.

Reply:
That was my mistake counting tiles I guess. This is my original draft when I worked on this:
![image](https://github.com/linz/argo-tasks/assets/86932794/a0175a21-c1dc-4d22-b2e2-2f951a059492)